libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
Always I would assume, would that be an issue? There is one advantage to testing.go: This would also work across packages. I have this issue for example in metricbeat. Several sub packages need the same test helper, so putting it in support_test.go does not work, as only the test files in the same package are compiled. That is why the test helper ended up in the main file: https://github.com/elastic/beats/blob/master/metricbeat/module/redis/redis.go